index.html

```diff
@@ -30,8 +30,20 @@
           </div>
         </div>
       </nav>
+      <!-- Starting mobile menu  -->
+      <div class="mobile-menu">
+        <div class="mobile-menu-header">
+        </div>
+        <ul class="mobile-menu-items">
+          <li><a href="#portfolio" class="mobile-header-color">Portfolio</a></li>
+          <li><a href="#about" class="mobile-header-color">About</a></li>
+          <li><a href="#contact" class="mobile-header-color">Contact</a></li>
+        </ul>
+                  <div class="close-btn">&#10005;</div>
+      </div>
```

Comment:
| Yours | Figma :+1:  |
| --- | --- |
| ![Screenshot from 2023-03-17 08-51-07](https://user-images.githubusercontent.com/63087491/225846185-fa0aa855-c081-4069-80c5-b1783a226d7e.png) | ![Screenshot from 2023-03-17 08-55-45](https://user-images.githubusercontent.com/63087491/225846217-25097742-7651-457e-a46e-f2947e02ba19.png) |

- [ ] Hi @omer-creator looking at the screenshots above you'll notice that there's a little difference between your design and that of Figma. We need to adjust the font weight of our close-btn and make sure that the vertical distance between the close button and our menu items is the same as that of Figma. You'll kindly required to increase the font weight of the menu items.

Reply:
Hey, I tried to resolve this issue and it looks same, please confirm it and let me know if it still requires some changes

Reply:
- [ ] You still have a couple of things to rectify before this gets approved. Kindly increase the `font-size` of your `nav-items` , and also use the correct `margin-left` and `margin-bottom` values in order to be consistent with the Figma design.
- [ ] Please notice that you now have horizontal scrolling which wasn't there prior to your most recent commit. Please try to fix it 👍 
### Figma Design
![image](https://user-images.githubusercontent.com/63087491/225846217-25097742-7651-457e-a46e-f2947e02ba19.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/225959173-94f76fd5-f6e3-43fb-951f-b6117051b66e.png)